LICENSE.txt

```diff
@@ -27,14 +27,14 @@ met:
    documentation and/or other materials provided with the
    distribution.
 
- * Neither the name of Kitware nor the names of any contributors may
+ * Neither the name of the copyright holder nor the names of its contributors may
```

Comment:
Note that this is a set of minimal changes to minimize diff. I did not use the numbers from each clause from the OSI text.